src/usr/share/netcheck/netcheck.py

```diff
@@ -108,6 +113,7 @@ def start(self):
         """
         try:
             self.network_helper.update_available_connections()
+        #pylint: disable=broad-except
```

Comment:
It looks like this applies to the current "block", so if you want to disable this for the entire file, just put one on the top.

Alternatively, you can put it in a configuration file.